install-opentelemetry-dotnet.ps1

```diff
@@ -0,0 +1,163 @@
+    } else {
+        throw "TODO: Not supported yet."
+    }
+
+    $dlUrl = "https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/download/$Version/$archive"
+    $dlPath = Join-Path $Path $archive
+    Invoke-WebRequest -Uri $dlUrl -OutFile $dlPath
+
+    return $dlPath
+}
+
+function Extract-OpenTelemetry([string]$DlPath, [string] $InstallPath) {
+    Expand-Archive $DlPath $InstallPath -Force
+}
+
+function Setup-OpenTelemetry-Environment([string] $OS, [string] $InstallPath) {
+    $target = [System.EnvironmentVariableTarget]::Machine
+
+    if($OS -eq "windows") {
+        # .NET Framework
+        [System.Environment]::SetEnvironmentVariable('COR_PROFILER','{918728DD-259F-4A6A-AC2B-B85E1B658318}', $target)
+        [System.Environment]::SetEnvironmentVariable('COR_PROFILER_PATH_32', (Join-Path $InstallPath "/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll"), $target)
+        [System.Environment]::SetEnvironmentVariable('COR_PROFILER_PATH_64', (Join-Path $InstallPath "/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll"), $target)
+
+        # .NET Core
+        [System.Environment]::SetEnvironmentVariable('CORECLR_PROFILER','{918728DD-259F-4A6A-AC2B-B85E1B658318}', $target)
+        [System.Environment]::SetEnvironmentVariable('CORECLR_PROFILER_PATH_32', (Join-Path $InstallPath "/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll"), $target)
+        [System.Environment]::SetEnvironmentVariable('CORECLR_PROFILER_PATH_64', (Join-Path $InstallPath "/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll"), $target)
+    }
+
+    # .NET Common
+    [System.Environment]::SetEnvironmentVariable('DOTNET_ADDITIONAL_DEPS', (Join-Path $InstallPath "AdditionalDeps", $target))
+    [System.Environment]::SetEnvironmentVariable('DOTNET_SHARED_STORE', (Join-Path $InstallPath "store", $target))
+    [System.Environment]::SetEnvironmentVariable('DOTNET_STARTUP_HOOKS', (Join-Path $InstallPath "netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll"), $target)
+
+    # ASP.NET Core
+    [System.Environment]::SetEnvironmentVariable('ASPNETCORE_HOSTINGSTARTUPASSEMBLIES', "OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper", $target)
+
+    # OpenTelemetry
+    [System.Environment]::SetEnvironmentVariable('OTEL_DOTNET_AUTO_HOME', $InstallPath, $target)
+    [System.Environment]::SetEnvironmentVariable('OTEL_DOTNET_AUTO_INTEGRATIONS_FILE', (Join-Path $InstallPath "integrations.json"), $target)
+}
+
+$Version = "v0.3.1-beta.1"
+$CLIArchitecture = Get-CLIArchitecture-From-Architecture $Architecture
+$InstallDir = Get-Install-Directory $OS $CLIArchitecture
+$TempDir = Get-Temp-Directory $OS
+
+Write-Output $OS
+Write-Output $CLIArchitecture
+Write-Output $InstallDir
+Write-Output $TempDir
+
+$DlPath = Download-OpenTelemetry $OS $Architecture $Version $TempDir
+Prepare-Install-Directory $InstallDir
+
+Extract-OpenTelemetry $DlPath $InstallDir
+Setup-OpenTelemetry-Environment $OS $InstallDir
```

Comment:
Can we do 2 scripts?
One for downloading and the second for "instrumenting"?
The user does not need to download it each time.

Reply:
I think the second one about instrumenting should go to readme?